x-pack/plugins/fleet/common/types/rest_spec/common.ts

```diff
@@ -7,6 +7,10 @@
 
 import type { HttpFetchQuery } from '@kbn/core/public';
 
+/**
+ * @deprecated
+ */
+// Deprecation - will be replaced by a "narrow" set of parameters
```

Comment:
sorry, I meant to move this inside the tsdoc like:

```
/**
 * @deprecated will be replaced by a "narrow" set of parameters
 */
```

Reply:
oh I didn't know it could be done like this! Thanks Julia!